src/Tools/Source/CompilerGeneratorTools/Source/IOperationGenerator/IOperationClassWriter.cs

```diff
@@ -148,21 +147,6 @@ private void WriteFiles()
                 }
             }
 
-            using (_writer = new StreamWriter(File.Open(Path.Combine(_location, "OperationKind.Generated.cs"), FileMode.Create)))
-            {
-                writeHeader();
-                WriteUsing("System");
-                WriteUsing("System.ComponentModel");
-                WriteUsing("Microsoft.CodeAnalysis.FlowAnalysis");
-                WriteUsing("Microsoft.CodeAnalysis.Operations");
-
-                WriteStartNamespace(namespaceSuffix: null);
-
-                WriteOperationKind();
```

Comment:
@333fred So if this is being removed, where it's generated?

Reply:
https://github.com/dotnet/roslyn/blob/main/src/Tools/Source/CompilerGeneratorTools/Source/IOperationGenerator/IOperationClassWriter.cs#L283.

Reply:
Yes, but that's not called anywhere. Is this intentional?

Reply:
Uh, no, it's not. Thanks for catching this, I'll go fix.

Reply:
https://github.com/dotnet/roslyn/pull/53812/